code/style.css

```diff
@@ -26,33 +71,86 @@ p {
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  
+}
+
+.deck {
+  border-style: groove;
+  opacity: 0.7;
+}
+
+.deck:hover {
+  opacity: 1;
+}
+
+/* Animation to flip the killer weapons vard */
+
+.imgflip {
+transform: rotateY(360deg);
+transition: transform 0.8s;
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: black;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  opacity: 0.7;
+  border-style: groove;
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
+  background: black;
 
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  opacity: 0.7;
+  border-style: groove;
 }
 
 button {
+  font-family: "Courier New", Courier, monospace;
+  color: white;
   padding: 8px 16px;
-  background: crimson;
-  border: none;
+  background: none;
+  border: 5px solid white;
   font-size: 20px;
   border-radius: 8px;
   display: block;
   margin: 0 auto;
+  margin-top: 15px;
+}
+
+button:hover {
+  color: black;
+  background: white;
+}
+
+button:active {
+  transform: translateY(4px);
+  outline: 0;
+}
+
+/* Styles for loading animation */
+
+#myProgress {
+  display: none;
+  width: 100%;
+  background-color: #ddd;
+  margin-top: 50px;
+  opacity: 0.5
+}
+
+#myBar {
```

Comment:
That is great job!
Nonetheless, style it by class, not by id

Reply:
Needs to be solved

Reply:
fixed